velox/common/memory/MemoryUsageTracker.cpp

```diff
@@ -83,7 +83,7 @@ void MemoryUsageTracker::release() {
     }
     minReservationBytes_ = 0;
   }
-  if (freeable > 0) {
+  if (freeable != 0) {
```

Comment:
same question

Reply:
ditto. Just make the code to be consistent with previous version and fix this later.